src/integrate/scarches/config.vsh.yaml

```diff
@@ -0,0 +1,140 @@
+functionality:
+  name: scarches
+  namespace: "integrate"
+  version: "dev"
+  description: "Performs reference mapping with scArches"
+  authors:
+    - name: Vladimir Shitov
+      email: vladimir.shitov@helmholtz-muenchen.de
+      roles: [ author ]
+      props: { github: vladimirshitov, orcid: "0000-0002-1960-8812" }
+  argument_groups:
+    - name: Inputs
+      arguments:
+        - name: "--input"
+          alternatives: ["-i"]
+          type: file
+          description: Input h5mu file to use as a query
+          direction: input
+          required: true
+        - name: "--modality"
+          type: string
+          default: "rna"
+          required: false
+        - name: "--reference"
+          choices: ["HLCA"]
+          alternatives: ["-r"]
+          type: string
+          description: Name of the reference dataset
+          required: false
+        - name: "--unlabeled_category"
+          type: string
+          description: "Value used for unlabeled cells"
+          default: "Unknown"
+          required: false
+        - name: "--labels_key"
```

Comment:
You can get this from the HLCA model directly:
* check latest scvi-tools version for model loading functionality without data
-> can then check required fields via DataRegistry